### ISSUE TITLE
Update Azure AD Application Owner or Add Owner

### ISSUE THREAD

**Issue by mferris2421** [Azure/azure-powershell#9132]
## Description of the new feature

Updating ownership of an Azure AD application or adding an owner. The command currently exists in Azure AD powershell but not in the az module. It should be added to az if the az module is the future state of Azure powershell.

## Proposed implementation details (optional)

Either through the update-azadapplication cmdlet or through a new cmdlet

**Comment by markcowl** [comment]
@mferris2421 Thanks for the good suggestion.  The Azure AD cmdlets in Azure PowerShell are not currently meant as a replacement for all of the functionality in the AzureAD module, and this functionality is not supported by the REST API that underlies these cmdlets, currently.  However, we are looking at releasing an updated version based on a later API this year, and we would definitely ad this cmdlet if the newer API supports it.

**Comment by Damag3d** [comment]
> Azure PowerShell are not currently meant as a replacement for all of the functionality in the AzureAD module

Why? Nobody needs dysfunctional AzureAD that works only on outdated PS5.1, needs desktop version, has some strange dependencies on Windows Forms and still does not cover all functionality compared to Az CLI/ARM templates. AzureAD must die.

**Comment by daveneeley** [comment]
This is different from #8412 then? Or would it be resolved by it?

**Comment by karok2m** [comment]
@daveneeley, no, https://github.com/Azure/azure-powershell/issues/8412 does not seem to address this problem. Proposed cmdlet should be similar to [Add-AzureADApplicationOwner](https://docs.microsoft.com/powershell/module/azuread/add-azureadapplicationowner?view=azureadps-2.0).
@markcowl, are there any updates on the topic ?

**Comment by Kwinno** [comment]
@markcowl Do you have any update?
We run an environment with 100s applications and this just does not scale well.

**Comment by MaddyMicrosoft** [comment]
Az.Resources exposes owner cmdlets for groups (Get-/New-/Remove-AzADGroupOwner) but not for applications or service principals. Adding them isn't on the current roadmap, but the broader ask for app/SP owner cmdlets in Az is being tracked in #16184.

To manage application owners today, use the Microsoft Graph PowerShell SDK. Closing on that basis. If this becomes blocking, please add to #16184 or open an issue in the MS Graph SDK repo: https://github.com/microsoftgraph/msgraph-sdk-powershell
